CactbotOverlay/FFXIVProcess.cs

```diff
@@ -512,6 +573,16 @@ public class EntityData {
             data.debug_job += string.Format("{0:x2}", job_bytes[i]);
           }
         }
+
+        for (int i = 0; i < data.statusEffects.Length; i++) {
+          var currentBase = kEntityStructureOffsetCharacterDetails + kEntityStructureOffsetStatusEffects + i * StatusEffect.structSize;
+
+          var id = BitConverter.ToUInt16(bytes, currentBase + StatusEffect.offsetId);
+          var stackCount = bytes[currentBase + StatusEffect.offsetStackCount];
+          var duration = BitConverter.ToSingle(bytes, currentBase + StatusEffect.offsetDuration);
```

Comment:
Oh wow, these are integers and not milliseconds remaining like on the job gauge.

Reply:
It is actually a single precision float

Reply:
When I printed these out, they sure looked like integers.  Are they getting converted at some point?